lib/twilio-ruby/framework/error.rb

```diff
@@ -11,12 +11,12 @@ def body
     end
 
     class RestError < TwilioError
-      attr_reader :message, :response, :code, :status_code, :detail, :more_info, :error_message
+      attr_reader :message, :response, :code, :status_code, :details, :more_info, :error_message
```

Comment:
yeah that was the concern behind removing `detail`, removing this will also make it compatible with definitions of `RestException` in other libs. Thoughts ?

Reply:
I'm okay with the change. AFAIK, `detail` was always `nil` anyway.